sdk/ml/azure-ai-ml/azure/ai/ml/entities/_assets/_artifacts/_package/inferencing_server.py

```diff
@@ -0,0 +1,160 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# pylint: disable=protected-access,redefined-builtin
+
+from azure.ai.ml._restclient.v2023_02_01_preview.models import (
+    AzureMLOnlineInferencingServer as RestAzureMLOnlineInferencingServer,
+    AzureMLBatchInferencingServer as RestAzureMLBatchInferencingServer,
+    CustomInferencingServer as RestCustomInferencingServer,
+    TritonInferencingServer as RestTritonInferencingServer,
+    OnlineInferenceConfiguration as RestOnlineInferenceConfiguration,
+    Route as RestRoute,
+)
+from azure.ai.ml._utils._experimental import experimental
+
+
+@experimental
+class AzureMLOnlineInferencingServer:
+    def __init__(self, type: str = None, code_configuration: str = None):
+        """Azure ML online inferencing configurations.
+
+        :param type: The type of the inferencing server.
+        :type type: str
+        :param code_configuration: The code configuration of the inferencing server.
+        :type code_configuration: str
+        """
+        self.type = type
+        self.code_configuration = code_configuration
+
+    @classmethod
+    def _from_rest_object(cls, rest_obj: RestAzureMLOnlineInferencingServer) -> "RestAzureMLOnlineInferencingServer":
+        return AzureMLOnlineInferencingServer(type=rest_obj.server_type, code_configuration=rest_obj.code_configuration)
+
+    def _to_rest_object(self) -> RestAzureMLOnlineInferencingServer:
+        return RestAzureMLOnlineInferencingServer(server_type=self.type, code_configuration=self.code_configuration)
+
+
+@experimental
+class AzureMLBatchInferencingServer:
+    def __init__(self, type: str = None, code_configuration: str = None):
```

Comment:
Why is "type" exposed here and in OnlineInferencingServer?
Isn't the type a constant at this point since we created classes for each inference server type?

Reply:
It should be exposed for all the classes of this kind, it was missing for Triton, I have fixed, it, these are exposed as private preview and will go through iteration and will have more certainty